recipes/conversation_assistant/config.default.js

```diff
@@ -1,5 +1,5 @@
 export default {
-    assistantId: '', // add your assistant id from Watson Assistant
+    assistantId: '', // add your environment id from Watson Assistant
```

Comment:
should we rename `assistantId` to `environmentId` since that's the new language?